### PR TITLE
SOLR-13394: Change default GC from CMS to G1

### PR DESCRIPTION
**Pull request by kesharee**

### CODE REVIEW

File: solr/bin/solr.cmd [hunk 1]
```diff
@@ -1167,20 +1167,14 @@ set SOLR_OPTS=%SOLR_JAVA_STACK_SIZE% %SOLR_OPTS%
...
-   -XX:+ParallelRefProcEnabled ^
-   -XX:-OmitStackTraceInFastThrow
+  set GC_TUNE=-XX:+UseG1GC ^
+    -XX:+PerfDisableSharedMem ^
+    -XX:+ParallelRefProcEnabled ^
+    -XX:G1HeapRegionSize=16m ^
+    -XX:MaxGCPauseMillis=250 ^
+    -XX:InitiatingHeapOccupancyPercent=45 ^
+    -XX:+UseLargePages ^
+    -XX:+AggressiveOpts
```
Comment (uschindler) [inline solr/bin/solr.cmd]: Don't enable aggressive opts!

Comment (kesharee) [reply solr/bin/solr.cmd]: Right, removing the experimental feature, Its deprecated in latest java versions

**Comment by risdenk** [comment]
@chatman - I think this PR can be closed?

**Comment by ctargett** [comment]
Since SOLR-13394 is resolved as of 8.1, I assume this can be closed.

**Pull request closed without merging**